index.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Microverse Survay Form</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
```

Comment:
You need add the "viewport" meta data as part of the best practices working with HTML and CSS